Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py

```diff
@@ -0,0 +1,2138 @@
+
+        if 'indicator_observations' in ind:
+            if len(ind['indicator_observations']) >= 0:
+                context['TC.Indicator(val.ID && val.ID === obj.ID)'][0]['IndicatorsObservations'] = ind[
+                    'indicator_observations']
+
+    context = {k: createContext(v, removeNull=True)[:MAX_CONTEXT] for k, v in context.items() if len(v) > 0}
+    return context, context.get('TC.Indicator(val.ID && val.ID === obj.ID)', [])
+
+
+def get_xindapi(indicator_value, indicatorType, owner, tc):
+    stdout = []
+    types = tc_get_indicator_types_request()['data']['indicatorType']
+    if indicatorType:
+        for item in types:
+            if item['apiEntity'] == indicatorType:
+                apiBranch = item['apiBranch']
+                ro = RequestObject()
+                ro.set_http_method('GET')
+                ro.set_owner(owner)
+                ro.set_request_uri('/v2/indicators/' + str(apiBranch) + "/" + quote(indicator_value).replace("/", "%2F"))
+                demisto.log('/v2/indicators/' + str(apiBranch) + "/" + quote(indicator_value).replace("/", "%2F"))
+                results = tc.api_request(ro)
+                if results.headers['content-type'] == 'application/json':
+                    if results.json()['status'] == 'Success':
+                        res = results.json()['data'][item['apiEntity']]
+                        res['owner'] = res['owner']['name']
+                        res['type'] = item['name']
+                        stdout.append(res)
+                        break
+    else:
+        for item in types:
+            if indicatorType and item['apiEntity'] == indicatorType:
+                apiBranch = item['apiBranch']
+            else:
+                apiBranch = item['apiBranch']
+            ro = RequestObject()
+            ro.set_http_method('GET')
+            ro.set_owner(owner)
+            ro.set_request_uri('/v2/indicators/' + str(apiBranch) + "/" + quote(indicator_value).replace("/", "%2F"))
+            results = tc.api_request(ro)
+            if results.headers['content-type'] == 'application/json':
+                if results.json()['status'] == 'Success':
+                    res = results.json()['data'][item['apiEntity']]
+                    res['ownerName'] = res['owner']['name']
+                    res['type'] = item['name']
+                    stdout.append(res)
+                    break
+
+    return stdout
+
+
+def get_indapi(indicator_value, owner, tc):
+    ro = RequestObject()
+    ro.set_http_method('GET')
+    ro.set_owner(owner)
+    ro.set_request_uri('/v2/indicators?filters=summary/v2/indicators?filters=summary%3D'
+                       + quote(indicator_value).replace("/", "%2F"))
+    results = tc.api_request(ro)
+    if results.headers['content-type'] == 'application/json':
+        return results.json()
+    else:
+        return {}
+
+
+def get_indicatorOwner(indicator_value, owner=demisto.params()['defaultOrg']):
```

Comment:
1) function name should be snake_case
2) don't call a function as default value
```suggestion
def get_indicator_owner(indicator_value, owner=None):
    owner = demisto.params()['defaultOrg'] if owner is None else owner
```